beamer/agent.py

```diff
@@ -51,6 +51,10 @@ def __init__(self, config: Config):
         request_manager = l2a_contracts["RequestManager"]
         fill_manager = l2b_contracts["FillManager"]
 
+        max_validity_period = request_manager.functions.MAX_VALIDITY_PERIOD().call()
+
+        if config.unsafe_fill_time > max_validity_period:
+            raise RuntimeError(f"Unsafe fill time should not be bigger than {max_validity_period}")
```

Comment:
We should probably use `>=` instead of `>` because if these two times are equal, the agent will not fill any requests.

I would also suggest replacing `should not` with `must not` in the exception message.